packages/v4/src/content/design-guidelines/usage-and-behavior/modal/modal.md

```diff
@@ -32,25 +43,33 @@ Modal dialogs interrupt user task flow and should be avoided for any information
 
 ### Types of modals
 #### Get confirmation
-Use a confirmation dialog to validate user decisions and communicate consequences. Confirmation dialogs need to clearly restate the action the user is trying to take and explicitly outline the consequences of that action.
+Use a confirmation dialog to validate user decisions and communicate consequences. Confirmation dialogs need to clearly restate the action the user is trying to take and explicitly outline the consequences of that action. They should be clear, concise, and direct. Give the user the important information first—additional detail like “Are you sure?” clutters your message and can be distracting.
+
+Depending on your design template, follow this format for confirmation dialogs:
+* **Headline:** A headline is usually phrased as a question. Include important keywords (like “permanent”) in the headline.
+* **Body text:** Body text gives information about the action’s consequence.
+* **Buttons:** Buttons allow a user to answer the headline question. These can simply be “yes” or “no,” but adding some context increases clarity—especially when skimming.
 
 **Note:** Since confirmation dialogs disrupt user task flow, do not provide confirmation dialogs for actions that are easily reversed or insignificant.
 
 ##### Confirm a non-destructive action
-![Modal](./img/confirm-non-destructive-action.png)
+<img src="./img/confirm-non-destructive-action.png" alt="Confirm a non-destructive action" width="1100"/>
+
 Provide buttons on confirmation dialogs that make it simple for users to understand and confirm their choices. When choosing button labels, use descriptive verbs that clearly indicate the consequence of a selection, like Delete, Save, or Create. Avoid vague labels like OK or Done.
 
 Ambiguous title and button labels make it more difficult for the user to understand their options and make a decision. Descriptive title and button labels make it easy for the user to scan the content and make a decision quickly.
 
 ##### Confirm a destructive action
-![Modal](./img/confirm-destructive-action.png)
+<img src="./img/confirm-destructive-action.png" alt="Confirm a destructive action" width="1100"/>
+
 If an action is difficult or impossible to undo, a warning icon can be incorporated into the modal to add emphasis. If an action will seriously impact the system and cannot be undone, use a destructive style button. Otherwise, use a primary button.
 
 Sometimes it is necessary to require an additional confirmation with a destructive action. An example using a form field can be seen below. In order for the destructive action to become actionable, the user needs to type in a word or phrase to make the button active.
-![Modal](./img/2-step-confirm-destructive-action.png)
+<img src="./img/2-step-confirm-destructive-action.png" alt="2 step confirm a destructive action" width="1100"/>
```

Comment:
Maybe this example can be improved. This is a poor confirmation dialog because it never tells me the name of the thing I am going to delete. Sometime I've seen this done where the user needs to somehow type or select the name before Delete is enabled.

Reply:
@mcarrano I think the name in this case is "Database X"! But yes, maybe changing it to something else would make that more clear.